CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v0.7.0dev - [date]
```

Comment:
```suggestion
## V1.0.0dev - "Nickname"
```

Could already bump this to 1.0.0dev for testing purposes and add a nice codename ?

Reply:
We'll aim to keep the version of the nf-core/scflow pipeline in sync with the scFlow R package and the Docker file.  We'll bump this to 1.0.0dev soon and name it but would like to avoid an NF cache reset right now if that's ok? :)